### PR TITLE
Release/6.2.0

### PR DESCRIPTION
**Pull request by ifrit98**
* Adds streaming functionality to bittensor api in the dendrite. May subclass from `StreamingSynapse` or build bespoke and pass `streaming=True` to the dendrite when querying.
* `btcli s register` now defaults to `recycle_register` functionality to avoid confusion. PoW is now turned OFF for all subnets for now.
* `btcli s pow_register` is introduced to retain the functionality for subnets who will turn on PoW for registration
* Adds `btcli w balance` command to see balance across all coldkeys on a given wallets `path`.
* Normalize weight values in `btcli r get_weights`
* Automatically check in `metagraph` if we are in the root network, rather than passing a `root` flag.
* Remove wallet `reregister` as PoW is turned off
* Limits `btcli w faucet` to max 3 iterations per command. (This is turned OFF on testnet atm)
* Other small bug fixes, etc.

### CODE REVIEW

**Review by Eugene-hu — APPROVED** [review]

**Review by Rubberbandits — COMMENTED** [review]
LGTM

**Review by Rubberbandits — APPROVED** [review]

File: bittensor/dendrite.py [hunk 1]
```diff
@@ -133,66 +246,112 @@ async def forward(
...
         """
-        Makes asynchronous requests to multiple target Axons and returns the server responses.
+        Asynchronously sends requests to one or multiple Axons and collates their responses.
+
+        This function acts as a bridge for sending multiple requests concurrently or sequentially
+        based on the provided parameters. It checks the type of the target Axons, preprocesses
+        the requests, and then sends them off. After getting the responses, it processes and
+        collates them into a unified format.
+
+        When querying an Axon that sends back data in chunks using the Dednrite, this function
```
Comment (p-ferreira) [inline bittensor/dendrite.py]: typo: `Dendrite` instead of `Dednrite`

File: bittensor/metagraph.py [hunk 2]
```diff
@@ -474,17 +473,15 @@ def _create_tensor(self, data, dtype) -> torch.nn.Parameter:
...
+    def _set_weights_and_bonds(self, subtensor: bittensor.subtensor = None):
         """
         Computes and sets weights and bonds for each neuron.
 
         Returns:
             None.
         """
         # TODO: Check and test the computation of weights and bonds
-        if root:
+        if self.netuid == 0:  # Is this the root network?
```
Comment (p-ferreira) [inline bittensor/metagraph.py]: Should this question be here in the code?

**Review by p-ferreira — APPROVED** [review]

File: bittensor/dendrite.py (hunk 1 above)
Comment (ifrit98) [reply bittensor/dendrite.py]: D-E-D DED.

File: bittensor/metagraph.py (hunk 2 above)
Comment (ifrit98) [reply bittensor/metagraph.py]: Prob not. :D
